pkgs/by-name/tz/tzf-rs/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tzf-rs";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "ringsaturn";
+    repo = "tzf-rs";
+    rev = "v${version}";
```

using finalAttrs would be better here
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "tzf-rs";
  version = "1.0.0";

  src = fetchFromGitHub {
    owner = "ringsaturn";
    repo = "tzf-rs";
    rev = "v${finalAttrs.version}";
```